pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -524,7 +527,9 @@ public static class QueryOptionKey {
 
         public static final String NUM_REPLICA_GROUPS_TO_QUERY = "numReplicaGroupsToQuery";
 
+        @Deprecated
         public static final String ORDERED_PREFERRED_REPLICAS = "orderedPreferredReplicas";
```

Comment:
This hasn't been released yet, any reason to not simply remove it instead of marking it as deprecated?

Reply:
I already cherry-pick this change inside Uber's infra for a couple of weeks.